welcome/oke_about.adoc

```diff
@@ -111,7 +111,7 @@ vulnerabilities and errors protection as {product-title}. {oke} includes a
 allows you to use an integrated Linux operating system with container runtime
 from the same technology provider.
 
-{oke} is subscription compatible with Windows Containers from Microsoft.
+{oke} is subscription compatible with {productwinc}.
```

Comment:
Not something you have to fix, but this sentence is goofy and probably should be reworded in the future. Though as we found I don't know if we have an OKE writer to ask :shrug:

Reply:
I agree entirely!